packages/config/src/projects/layer2s/zksyncera.ts

```diff
@@ -87,6 +88,13 @@ const validators = () => {
 export const zksyncera: Layer2 = {
   type: 'layer2',
   id: ProjectId('zksync2'),
+  badges: [
+    Badge.VM.EVM,
+    Badge.DA.EthereumBlobs,
+    Badge.Stack.ZKStack,
+    Badge.Other.Governance,
+    Badge.Other.AccountAbstraction,
```

Comment:
i wouldnt put account abstraction badge before we first define it and then properly assess everyone